recipes/anyrpc/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(test_anyrpc CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(anyrpc REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} ../test_package/test_anyrpc.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE anyrpc::anyrpc)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

You can cut down on duplication here: https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/test_v1_package/CMakeLists.txt

```suggestion
cmake_minimum_required(VERSION 3.1)
project(test_package)

include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup(TARGETS)

add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package/
                 ${CMAKE_CURRENT_BINARY_DIR}/test_package/)
```

---

done